pkgs/misc/cups/default.nix

```
@@ -41,6 +41,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" "dev" "man" ];
 
+  patches = lib.optional (version == "2.2.6") ./0001-TargetConditionals.patch;
```

```suggestion
  patches = lib.optional stdenv.isDarwin ./0001-TargetConditionals.patch;
```
So we do not forget it that easily.

---

I can add a comment but it's more accurate when applied depending on the version.

---

Thats totally true but with this the patch will most likely fail when updating and we do not forget it.